Framework/CurveFitting/inc/MantidCurveFitting/Algorithms/LeBailFit.h

```diff
@@ -267,9 +272,9 @@ class DLLExport LeBailFit : public API::Algorithm {
   //--------------  Background function Suite
   //----------------------------------------
   /// Re-fit background according to the new values
-  void fitBackground(size_t wsindex, API::FunctionDomain1DVector domain,
-                     API::FunctionValues values,
-                     std::vector<double> &background);
+  // std::vector<double> fitBackground(size_t wsindex,
+  //                                  API::FunctionDomain1DVector domain,
+  //                                  API::FunctionValues values);
```

Comment:
I was tempted to remove it, but I didn't in the end because of [this](https://github.com/mantidproject/mantid/blob/17710_HistogramData_Rollout_Algorithms_76-81_new/Framework/CurveFitting/src/Algorithms/LeBailFit.cpp#L2413) comment. I don't know if anyone is planning to do work on this... Methods `LeBailFit::smoothBackgroundExponential` and `LeBailFit::smoothBackgroundAnalytical` also contain dead code which I didn't remove.